mesos_master/assets/service_checks.json

```diff
@@ -5,13 +5,14 @@
         "check": "mesos_master.can_connect",
         "statuses": [
             "ok",
-            "critical"
+            "critical",
+            "unknown"
         ],
         "name": "Can Connect",
         "groups": [
             "host",
             "url"
         ],
-        "description": "Returns CRITICAL if the Agent cannot connect to the Mesos Master API to collect metrics, otherwise OK."
+        "description": "Returns CRITICAL if the Agent cannot connect to the Mesos Master API to collect metrics, UNKNWON if the master is not detected as the leader, otherwise OK."
```

Comment:
```suggestion
        "description": "Returns CRITICAL if the Agent cannot connect to the Mesos Master API to collect metrics, UNKNOWN if the master is not detected as the leader, otherwise OK."
```